torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2599,7 +2599,8 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            # BFloat16 support on CUDA requires CUDA 11 and SM53
            dtypesIfCUDA=floating_types_and(torch.float16, torch.complex64, torch.complex128,
                                            *[torch.bfloat16] if CUDA11OrLater else []),
-           dtypesIfROCM=floating_types_and(torch.half),
+           dtypesIfROCM=floating_types_and(torch.float16, torch.complex64, torch.complex128,
+                                           *[torch.bfloat16] if CUDA11OrLater else []),
```

Comment:
Does this condition `if CUDA11OrLater else []` make sense for ROCm?

Reply:
Sorry for that. Removed the check and just added bfloat16.